### ISSUE TITLE
What are Quilt and Stacked Git?

### ISSUE THREAD

**Issue by bradym** [jwiegley/git-from-the-bottom-up#23]
On https://jwiegley.github.io/git-from-the-bottom-up/2-The-Index/2-taking-the-index-further.html Quilt and Stacked Git are mentioned without explanations of what they are or links to learn more about them. Based on context, I assume that `stg` _is_ Stacked Git, but this is also a bit unclear.

**Comment by Paciupa** [comment]
Yes, I also expected some explanations about what stg is, but there is good documentation for [Stacked Git](https://stacked-git.github.io/guides/usage-example/)
